lib/ui/widgets/appSelectorView/installed_app_item.dart

```diff
@@ -52,26 +51,25 @@ class _InstalledAppItemState extends State<InstalledAppItem> {
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: <Widget>[
-                  Row(
-                    children: [
-                      Text(
-                        widget.name,
-                        maxLines: 2,
-                        overflow: TextOverflow.visible,
-                        style: const TextStyle(
-                          fontSize: 16,
-                          fontWeight: FontWeight.w500,
-                        ),
-                      ),
-                      const SizedBox(width: 4),
-                      I18nText(
-                        widget.installedVersion.isEmpty
-                            ? 'All versions'
-                            : 'v${widget.installedVersion}',
-                      ),
-                    ],
+                  Text(
```

Comment:
Why was the Row removed? Semantically each line of text should be a row item

Reply:
That's because the installed version is shown right above the suggested version and no Row is needed.

Reply:
What if the device is widescreen? Won't the items stop to be wrapped and showed next to each other? Using Row and Columns usually also are also done for semantical reasons. If you know that you want to show a line of text below another, you use a row of those items

Reply:
I could have used Wrap as well, but I guess the Installed version and the Suggested version should be next to each other.